Gemfile.lock

```diff
@@ -206,8 +206,9 @@ GEM
       actionview (>= 5.0.0)
       activesupport (>= 5.0.0)
     jmespath (1.6.1)
-    jsbundling-rails (1.0.2)
+    jsbundling-rails (1.0.3)
```

Comment:
### [jsbundling-rails](https://github.com/rails/jsbundling-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.2 → 1.0.3

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/jsbundling-rails/1.0.2/1.0.3)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/rails/jsbundling-rails/compare/v1.0.2...v1.0.3).

These are the individual commits:
- (493cd94) [Remove needless `webpack-cli` installing (#96)](https://github.com/rails/jsbundling-rails/commit/493cd94c76e16358cd8f92db3bf640c21bd29f38)
- (fa1ed8a) [Update comparison_with_webpacker.md (#79)](https://github.com/rails/jsbundling-rails/commit/fa1ed8a00ac474479902931db61f855fb436efea)
- (fc66837) [Use bundle add instead (#103)](https://github.com/rails/jsbundling-rails/commit/fc668372a2bab8d289396f4551c2aec5b11c53fa)
- (5e6758f) [Adding --public-path option to the esbuild script (#58)](https://github.com/rails/jsbundling-rails/commit/5e6758f077f0be59f94ea34678c2f8c452f2ba32)
- (d7ab4f4) [Removes js maps in assets clobber task (#110)](https://github.com/rails/jsbundling-rails/commit/d7ab4f4a15aa1a2641c2d5224ec9cb89c4ba59fb)
- (8faaef0) [Update switch_from_webpacker.md (#109)](https://github.com/rails/jsbundling-rails/commit/8faaef0656a389b944bdf27e046c15e4c9df5e91)
- (90833fb) [Fix for foreman existence test in rbenv installs. (#113)](https://github.com/rails/jsbundling-rails/commit/90833fba708fb415447492fad6d6d2b6961ee5aa)
- (0c8b321) [Name output sourcemaps after the output file with its extens…](https://github.com/rails/jsbundling-rails/commit/0c8b3212d9ec29e24b45c1ec3d2a73ffb67f3997)
- (e7e5ad5) [Bump nokogiri](https://github.com/rails/jsbundling-rails/commit/e7e5ad516b4f805215007d8724d4d9ec7a496575)
- (1dbf90a) [Bump version for 1.0.3](https://github.com/rails/jsbundling-rails/commit/1dbf90a82898600ca900c56741dab3b4fb214015)


</details>